content/en/events/gsoc-2024.md

```diff
@@ -5,25 +5,53 @@ description = "Google Summer of Code 2024"
 
 ---
 
-Kubeflow Community is excited to announce that we are applying this year to participate in the [**Google Summer of Code 2024**](https://buildyourfuture.withgoogle.com/programs/summer-of-code)
+Kubeflow Community is excited to announce that we have been [selected](https://summerofcode.withgoogle.com/programs/2024/organizations/kubeflow) as organization to participate in [**Google Summer of Code 2024**](https://buildyourfuture.withgoogle.com/programs/summer-of-code).
 
-Below are the "ideas/features" from different projects in the Kubeflow that being submitted for seeking proposals from GSoC Students.
+Below are the "ideas/features" from different projects in Kubeflow that are seeking proposals from GSoC Students
```

Comment:
Its probably not necessary to say this, and putting things in quotes looks a bit strange.

```suggestion
This page aims to help you participate in the Kubeflow organization for GSoC 2024.
```